ups/eupspkg.cfg.sh

```diff
@@ -1 +1,20 @@
 export SCONSFLAGS=$SCONSFLAGS" USE_UNKNOWN_VARS=true WITH_BLAS=false PREFIX="$PREFIX
+install()
+{
+	default_install
+	if [[ $OSTYPE == darwin* ]]; then
+		curdir=`pwd`
+		cd $PREFIX/lib
+		rm libtmv.dylib
+		tmv_name=`ls libtmv.*.dylib`
+		tmv_symband_name=`ls libtmv_symband.*.dylib`
+		ln -s $tmv_name libtmv.dylib
+		install_name_tool -id @rpath/$tmv_name $tmv_name
+		rm libtmv_symband.dylib
+		ln -s $tmv_symband_name libtmv_symband.dylib
+		install_name_tool -id @rpath/$tmv_symband_name $tmv_symband_name
+		install_name_tool -change $PREFIX/lib/$tmv_name @rpath/$tmv_name $tmv_symband_name
```

Comment:
Why is both creating the symlinks and running install_name_tool necessary?

Reply:
the binaries have the path to their installed location burned into them. You have to use install_name_tool to strip that path out so that DYLD_LIBRARY_PATH will be allowed to work.